common/verify_checksum.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Suggestion: Do you want to combine wget also in this utility? Say, it takes two arguments - download link, and checksum? That way the caller scripts will look cleaner.